bigtable/client/row_reader.cc

```diff
@@ -117,6 +158,7 @@ void RowReader::MakeRequest() {
   if (rows_limit_ != NO_ROWS_LIMIT) {
     request.set_rows_limit(rows_limit_ - rows_count_);
   }
+  request.set_app_profile_id(app_profile_id_.get());
```

Comment:
Move this to the point where you call `set_table_name()`.  Should these use the `SetCommonTableOperationRequest` function here too?

Reply:
Done.